2-multitenant/modules/env_baseline/versions.tf

```diff
@@ -20,11 +20,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 6.6, < 7"
+      version = ">= 6.6, < 7, < 6.17"
```

Comment:
do you still need < 7 ? if there is < 6.17

Reply:
Technically no, but left it there to remind us once we can lift the `< 6.17` limit.

Reply:
sounds good